gcloud/_helpers.py

```diff
@@ -439,8 +441,7 @@ def _to_bytes(value, encoding='ascii'):
     :rtype: str / bytes
     :returns: The original value converted to bytes (if unicode) or as passed
               in if it started out as bytes.
-    :raises: :class:`TypeError <exceptions.TypeError>` if the value
-             could not be converted to bytes.
+    :raises TypeError: if the value could not be converted to bytes.
```

Comment:
I'd prefer something semantically richer, and that rendered well in Sphinx, over something that appeases pylint.  What if we ask the pylint team to add support for

```
:raises: :exc:`ValueError`
```